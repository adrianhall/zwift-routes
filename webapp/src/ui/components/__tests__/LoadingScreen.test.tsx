import React from 'react';
import { shallow } from 'enzyme';
import { LoadingScreen } from '..';

describe('LoadingScreen', () => {
  it('should render like the snapshot', () => {
    const component = shallow(<LoadingScreen />);
    expect(component).toMatchSnapshot()
  });
});
