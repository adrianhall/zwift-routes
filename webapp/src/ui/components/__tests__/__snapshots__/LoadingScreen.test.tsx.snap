// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen should render like the snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <Component
    color="#fff"
    loading={true}
    size={64}
    sizeUnit="px"
  />
</div>
`;
